test/cbmc/proofs/JSON_SearchConst/Makefile

```diff
@@ -0,0 +1,27 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
```

Comment:
Should `EXPENSIVE = true` be set considering that this proof takes upwards of 20GB per the safety-checks graph displayed on https://deh3ggah7odfx.cloudfront.net/final/ee34e1b2-bdad-4777-94cb-e02f32a590e1/index.html ?

Reply:
Implemented as suggested.